src/sentry/models/group.py

```diff
@@ -507,9 +507,11 @@ def filter_to_team(self, team):
 
         project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
         user_ids = list(team.member_set.values_list("user_id", flat=True))
-        assigned_groups = GroupAssignee.objects.filter(
-            Q(team=team) | Q(user_id__in=user_ids)
-        ).values_list("group_id", flat=True)
+
+        assigned_groups = GroupAssignee.objects.filter(team=team).values_list(
```

Comment:
i _think_ if you do

```
assigned_groups = (
    GroupAssignee.objects.filter(team=team) | 
    GroupAssignee.objects.filter(user_id__in=user_ids)
).values_list("group_id", flat=True)
```
it can do it in one query instead of two?